test/fixture/nuxt.config.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
     config: {
       debug: true
     },
+    cacheNames: {
```

Comment:
Can it be an array to allow specifying more than one cache name?

Reply:
I'm not sure if there's a need for it, as this sets the cache names application wide. For example, it allows you to set the default `workbox` cache name to something different. How could this be used with a array setup?

Reply:
OK, you are right. I checked the docs.